src/HomeWork.java

```diff
@@ -0,0 +1,91 @@
+
+public class HomeWork {
+    public static int calculateAbsoluteDifference(int a, int b) {
+        return Math.abs(a - b);
+    }
+
+    public static void main(String[] args) {
+        int a = 5;
+        int b = 9;
+        int result = calculateAbsoluteDifference(a, b);
+
+        System.out.println("a = " + a);
+        System.out.println("a = " + b);
+        System.out.println("result = " + result);
+
+        a = -8;
+        b = 7;
+        result = calculateAbsoluteDifference(a, b);
+
+        System.out.println("a = " + a);
+        System.out.println("a = " + b);
+        System.out.println("result = " + result);
+
+    }
+}
+
+class FirstChar {
```

Comment:
You don't need a new class for every task. You had to create everything in one class but with different methods